lib/utils.c

```diff
@@ -25,15 +25,15 @@
  * OTHER DEALINGS IN THE SOFTWARE.
  */
 
+#include "lib_common.h"
+
 #ifdef FREESTANDING
 #  define malloc NULL
 #  define free NULL
 #else
 #  include <stdlib.h>
 #endif
 
-#include "lib_common.h"
-
 #include "libdeflate.h"
```

Comment:
`libdeflate.h` should be moved above as well, in case it includes `stdlib.h` in the future.